app/helpers/gardens_helper.rb

```diff
@@ -9,6 +9,10 @@ def display_garden_description(garden)
     end
   end
 
+  def display_garden_name(garden)
+    truncate(garden.name, length: 50, separator: ' ', omission: '... ')
```

Comment:
http://www.w3schools.com/cssref/css3_pr_text-overflow.asp is also another option for preventing overflow issues which tends to work a bit better regardless of text size